visualization/visualization_test.go

```diff
@@ -97,6 +97,42 @@ func TestGenerateFlameGraphExecError(t *testing.T) {
 	mockExecutor.AssertExpectations(t)
 }
 
+func TestRunPerlScriptDoesExist(t *testing.T) {
+	mockOSWrapper := new(mockOSWrapper)
+	executor := defaultExecutor{
+		osWrapper: mockOSWrapper,
+	}
+	cwd, _ := os.Getwd()
```

Comment:
If `os.Getwd()` fails then runPerlScript is going to propagate the error when it calls `os.Getwd()`. Hence, this test will fail even if I don't check errors here.